### PR TITLE
vst2 and lv2

### LINKED ISSUE #13: ASIO CMake opt-in
Copy the rules we had in surge for the program here.

### PR DESCRIPTION
**Pull request by ghost**
adding the options
- vst2 works
- lv2 works
(todo: vst2 and lv2 show 64 I/O, thats a little too much
I'm guessing this has to do with #60)

Addresses #13

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,6 +75,10 @@ if (CLAP_JUCE_EXTENSIONS_ROOT)
           CLAP_ID "org.surge-synth-team.monique-monosynth")
 endif()
 
+if(JUCE_ASIO_SUPPORT)
+  target_compile_definitions(${PROJECT_NAME} PUBLIC JUCE_ASIO=1)
+endif()
```
Comment (baconpaul) [inline CMakeLists.txt]: Doesn’t this need to come after the env vst check? That is the asio is set later in the file so this should appear after that block? Or that block should appear before here?

Comment (ghost) [reply CMakeLists.txt]: yes :)

Comment (ghost) [reply CMakeLists.txt]: I removed the asio change from this pr, doesn't make much sense to push this untested.
need to set up a windows first

**Comment by baconpaul** [comment]
OK great!

I have a windows box I sometimes boot but also know @mkruselj is always down to help test a PR once its put back in place.

**Pull request merged**